cabal-install/Distribution/Client/Setup.hs

```diff
@@ -937,7 +953,12 @@ getCommand = CommandUI {
        ++ "the source\ntarball and unpacks it in a local subdirectory. "
        ++ "Alternatively, with -s it will\nget the code from the source "
        ++ "repository specified by the package.\n",
-    commandNotes        = Nothing,
+    commandNotes        = Just $ \pname ->
+          "Examples:\n"
+       ++ "  " ++ pname ++ " get hlint\n"
+       ++ "    Download the latest stable version of hlint;\n"
+       ++ "  " ++ pname ++ " get lens -shead\n"
+       ++ "    Download the source repository (i.e. git clone from github).\n",
```

Comment:
Perhaps we should use the long form for flags like this, `--source-repository=head`, or at least use a space `-s head`.